src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -207,41 +265,60 @@ class LocationDetails extends Component {
   render() {
     return (
       <div className="configuration-wizard-content flex-column">
+        <AddOrganizationModal
```

Comment:
move this inside the form, so you will have access to the mutators it, can be before `<form`, but inside `<Form`